src/compiler/commandLineParser.ts

```diff
@@ -376,6 +375,63 @@ namespace ts {
         }
     }
 
+    function parseMultiValueStringArray(s: string, existingValue: string[]) {

```

Comment:
as I can see this function parses comma separated non-quoted strings with assumption that value itself should not have commas inside, i.e. `"a,b", "c"`. Can you document this behavior (it is not completely clear otherwise by looking at `string[]` as an argument type)